src/components/tooltip/Tooltip.js.flow

```diff
@@ -74,212 +18,29 @@ type Props = {
     /** A CSS class for the tooltip */
     className?: string,
     /** Whether to constrain the tooltip to the element's scroll parent. Defaults to `false` */
-    constrainToScrollParent: boolean,
+    constrainToScrollParent?: boolean,
     /** Whether to constrain the tooltip to window. Defaults to `true` */
-    constrainToWindow: boolean,
+    constrainToWindow?: boolean,
     /** Forces the tooltip to be disabled irrespecitve of it's shown state. Defaults to `false` */
-    isDisabled: boolean,
+    isDisabled?: boolean,
     /** Forces the tooltip to be shown or hidden (useful for errors) */
     isShown?: boolean,
     /** Whether to add tabindex=0.  Defaults to `true` */
     isTabbable?: boolean,
     /** Function called if the user manually dismisses the tooltip - only applies if showCloseButton is true */
     onDismiss?: () => void,
     /** Where to position the tooltip relative to the wrapped component */
-    position: Position,
+    position?: Position,
```

Comment:
Shouldn't this be a required property of any tooltip

Reply:
`position` is already defined as part of the `defaultProps` in Tooltip.tsx, so components that use `Tooltip` do not have to provide it as a prop